### ISSUE TITLE
Query log empty

### ISSUE THREAD

**Issue by Windyo** [pi-hole/pi-hole#2198]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*.
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the the source code relevant to this issue?:**

`2`

---
**Expected behaviour:**

`Going on the Query page displays a list of queries`

**Actual behaviour:**

`No Data Available in Table`

**Steps to reproduce:**

`
* go to pihole
* click on query list`

**Debug token provided by [uploading `pihole -d` log](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug):**

`mdxmkh6ey2`

**Troubleshooting undertaken, and/or other relevant information:**

`
* revert to Master from FTL-DNS
* Check Query Logging is Enabled
* Pihole -r
* git fetch --tags && pihole -r
* pihole -up`

**Comment by Windyo** [comment]
Yeah I've already done that, but I just did it again.

I ran repair first, it didn't change anything, so i ran reconfigure.
here's the new token: szuf4bs65t

**Comment by Mcat12** [comment]
Run `sudo rm /etc/dnsmasq.d/01-pihole.conf` and run a repair.

**Comment by Windyo** [comment]
No luck, same symptoms.

new code j6dd0a2j1

**Comment by Mcat12** [comment]
Run:
```
sudo service dnsmasq restart
sudo service pihole-FTL restart
```

**Comment by Windyo** [comment]
no changes.

**Comment by Mcat12** [comment]
Make a new debug token

**Comment by Windyo** [comment]
tried to reinstall from scratch and ran into a new issue, see #2216
Putting this ticket on hold to see if it happens again after #2216 is closed.

**Comment by Windyo** [comment]
os back to this one: after reinstall, I get "FTL not installed" and "Lost connection to API" on the front page.

Seeing as I reinstalled from scratch I'm guessing there was some leftover from the initial install that makes pihole try to use FTLDNS ?

Anyway unless we find a solution i'm nuking the entire Pi from orbit.
As there was another posted with my issue tis seems to be related to FTL 3.0.


New token: m7ohmk96ij

**Comment by Windyo** [comment]
Nuked it from orbit, works again.
Probably an issue with FTL.
